common/src/provisioning/Authentication/AuthenticationMethodFactory.cs

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Microsoft.Azure.Devices.Common;
```

Comment:
This is moved into common - what else is referencing it?

Reply:
This class was not required for the sas token implementation; I have removed it